app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -226,8 +226,10 @@ protected void setSpoilerAndContent(boolean expanded,
                 setContentWarningButtonText(!expanded);
 
                 this.setTextVisible(sensitive, !expanded, content, mentions, tags, emojis, poll, statusDisplayOptions, listener);
+                mediaContainer.setVisibility(!expanded ? View.VISIBLE : View.GONE);
```

Comment:
Not quite sure though why this (and only this) would need a special handling here.